Formula/deno.rb

```diff
@@ -0,0 +1,35 @@
+class Deno < Formula
+  desc      "Secure TypeScript runtime on V8"
+  homepage  "https://github.com/denoland/deno/"
+  url       "https://github.com/denoland/deno.git",
+      :tag      => "v0.2.5",
+      :revision => "320577c131c8058ba0065f26119ce1ce18bc23d2"
+
+  bottle do
+  end
+
+  depends_on "ninja" => :build
+  depends_on "node" => :build
+  depends_on "python@2" => :build
+  depends_on "rust" => :build
+
+  def install
+    rm_rf ["node_modules", "buildtools", "build_overrides", "testing", "tools/clang"]
+    system "python", "tools/setup.py"
```

Comment:
According to https://github.com/denoland/deno/blob/master/.appveyor.yml this seems to download clang amongst other things. Can we use homebrew dependencies for that instead ?

Reply:
I'll investigate this.

Reply:
Having a depedency on `llvm` would be the most obvious choice however I'm not very familiar with the past history of the v8 formula and the decision that went in to creating that. Can you determine which llvm version is used/required and try and add that to the dependencies ?